github-config.yaml

```diff
@@ -3,6 +3,8 @@ orgs:
     admins:
       - durandom
       - goern
+      - tumido
+      - Shreyanand
```

Comment:
Please remove these them from members : https://github.com/open-services-group/community/blob/1c127b79924b05258003c1464a83b81848c7805a/github-config.yaml#L17
for this to function properly.